deep_qa/layers/tuple_matchers/slot_similarity_tuple_matcher.py

```diff
@@ -1,11 +1,11 @@
 from copy import deepcopy
-from typing import Any, Dict
+from typing import Dict, Any
```

Comment:
Nitpicking here, but alphabetizing these is better than not.  The line should stay how it was.